FilmsApp/View/GeneralController/CollectionListCell.swift

```diff
@@ -26,9 +29,16 @@ class MyCollectionViewCell: UICollectionViewCell {
         ])
     }
     
-    
-    
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    func setData(url: URL) {
+        DispatchQueue.global().async { [weak self] in
+            if let data = try? Data(contentsOf: url) {
+                        if let image = UIImage(data: data) {
+                            DispatchQueue.main.async {
+                                self?.myImageView.image = image
+                            }
+                        }
+                    }
+                }
```

Comment:
In order to avoid a lot of nestings, consider creating "data" and "image" in one "if" statement.